src/components/tx/SignOrExecuteForm/SignerForm/index.tsx

```diff
@@ -58,13 +58,14 @@ export const SignerForm = ({ willExecute }: { willExecute?: boolean }) => {
       return []
     }
     const owners = new Set(nestedSafeOwners ?? [])
+    const isFullySigned = safeTx ? safeTx.signatures.size >= safe.threshold : false
```

Comment:
Why only fully signed and not also `threshold - 1`?

Reply:
Because only fully signed transactions can be executed by non-owners.
There is another condition in case the connected wallet is an owner itself - in which case we always offer to chose it.